CONTRIBUTING.md

```diff
@@ -125,9 +125,10 @@ Please follow these steps to have your contribution considered by the maintainer
 
 1. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
 2. Follow the [styleguides](#styleguides)
-3. Assign the issue(s) to yourself to avoid concurent work on the same task.
-4. Open the PR as soon as possible in status `Draft` until it is ready for review.
-5. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+3. To avoid concurent work on the same task, assign the issue(s) to yourself or if you are _external contributor_, add a comment to ask @eviden-parallel/maintainers to be assigned the issue.
+4. Create a new branch for each pull request (PR) following the [branch naming convention](#branch-naming-convention). _External contributors_ should fork the repository and create the branch in [their fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+5. Open the PR as soon as possible in status `Draft` until it is ready for review.
```

Comment:
Not sure what you mean with WIP?
A draft PR already means it is work in progress and is not ready for review/merge...
Or do you have something else in mind?

Reply:
As discussed, i mean, if a commit is a work in progress, we could mark as such like:
```
fix: display validation errors properly (WIP), refs #100
feat: implement basic search (WIP), refs #101
chore: add correct german translation (WIP), refs #102
ci: repair live deployment (WIP), refs #103
refactor: unify URL formatting (WIP), refs #104
```
Source: https://dev.to/visuellverstehen/new-git-guidelines-we-have-switched-to-conventional-commits-1p0c

But as i said, we can skip that for sure!

Reply:
As discussed, lets skip WIP Topic for commits.

Reply:
Thanks for the clarifying discussion @juliankasimir-eviden 
To summarize:

- We should not use WIP in PRs instead of Draft because it would not be handled properly by either the CI (conditions) nor GitHub (e.g. notifications) nor in addtion to Draft since it would not be complient to convential commit requirements
- We should not use WIP in branch names, since all feature branches are basically work in progress until a PR is opened and merged after which the branch is deleted
- We should not use WIP for partial commits. [Commits should ideally be very granular and use specific messages that tell a story](https://github.blog/2022-06-30-write-better-commits-build-better-projects/)